src/libs/ReportActionsUtils.ts

```diff
@@ -324,7 +385,10 @@ function getCombinedReportActions(reportActions: ReportAction[], transactionThre
     const isSelfDM = report?.chatType === CONST.REPORT.CHAT_TYPE.SELF_DM;
     // Filter out request and send money request actions because we don't want to show any preview actions for one transaction reports
     const filteredReportActions = [...reportActions, ...filteredTransactionThreadReportActions].filter((action) => {
-        const actionType = (action as OriginalMessageIOU).originalMessage?.type ?? '';
+        if (!isMoneyRequestAction(action)) {
```

Comment:
@nkdengineer Coming from your new PR  https://github.com/Expensify/App/pull/48139, Why did we need to add this check?

Reply:
@DylanDylann It's only used to predict the type of action.